src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs

```diff
@@ -0,0 +1,18 @@
+namespace VolleyManagement.Data.MsSql.Context.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class initial : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.Tournaments", "City", c => c.String(maxLength: 50));
```

Comment:
Issue says field should be named location

Reply:
oops, sorry, I will change it